Week 4/profixNotationSolver/src/TwoOrMoreExpressionsException.java

```diff
@@ -0,0 +1,4 @@
+@SuppressWarnings("serial")
+public class TwoOrMoreExpressionsException extends Exception {
```

Comment:
naming is weird, it should be about remaining operand

Reply:
I changed it to NotAnExpressionException. Is it okay?